crates/nexmark/src/config.rs

```diff
@@ -90,40 +126,11 @@ pub struct Config {
     /// Specify the proportion of events that will be new people.
     #[clap(long, default_value = "1", env = "NEXMARK_PERSON_PROPORTION")]
     pub person_proportion: usize,
-
-    /// Dump DBSP profiles for all executed queries to the specified directory.
-    #[clap(long, env = "NEXMARK_PROFILE_PATH")]
```

Comment:
these clap things are not important?

Reply:
I moved them to another struct. Nothing was deleted (unless I made a mistake).

Reply:
found them